widgetssdk/src/main/java/com/glia/widgets/entrywidget/EntryWidgetController.kt

```diff
@@ -167,6 +167,7 @@ internal class EntryWidgetController @JvmOverloads constructor(
         val messaging = EntryWidgetContract.ItemType.Messaging(unreadMessagesCount)
 
         val items = queues.mediaTypes.mapNotNull {
+            println("*********************** authenticated: ${isAuthenticatedUseCase()}")
```

Comment:
@DavDo This has to be removed, isn't it?

Reply:
Gosh! yes!
Since this is already merged, we can force-push a fixed commit to the feature branch before merging it to do development.
Would you like me to do that?

Reply:
The feature branch is protected, as far as I know. Better to do it through a PR...